pkg/host/host_linux.go

```diff
@@ -17,12 +18,14 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/google/syzkaller/pkg/log"
 	"github.com/google/syzkaller/pkg/osutil"
 	"github.com/google/syzkaller/prog"
 	"github.com/google/syzkaller/sys/linux"
 )
 
 func isSupported(c *prog.Syscall, sandbox string) (bool, string) {
+	log.Logf(1, "checking support for %v", c.Name)
```

Comment:
Do we really want to log a line per syscall? Esp on level 1? It will be line 4000 lines of output. I think it will make level 1 hard to use.